inc/ThirdParty/Hostings/Cloudways.php

```diff
@@ -30,6 +30,26 @@ public static function get_subscribed_events() {
 		];
 	}
 
+	/**
+	 * Determine if the Varnish server is up and running.
+	 *
+	 * @since 3.6.1
+	 */
+	private static function is_varnish_running() {
+		if ( ! isset( $_SERVER['HTTP_X_VARNISH'] ) ) {
+			return false;
+		}
+
+		if ( ! isset( $_SERVER['HTTP_X_APPLICATION'] ) ) {
+			return false;
+		}
+
+		if ( 'varnishpass' === trim( strtolower( $_SERVER['HTTP_X_APPLICATION'] ) ) ) { // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
+			return false;
+		}
+		return true;
```

Comment:
No need to explicitly return `false` or `true` as the conditional expression returns a boolean.

```suggestion
		return ( 'varnishpass' !== trim( strtolower( $_SERVER['HTTP_X_APPLICATION'] ) ) )  // phpcs:ignore WordPress.Security.ValidatedSanitizedInput.InputNotSanitized, WordPress.Security.ValidatedSanitizedInput.MissingUnslash
```